src/inc/AppxPackaging.hpp

```diff
@@ -1479,6 +1479,13 @@ MSIX_API HRESULT STDMETHODCALLTYPE UnpackPackage(
     char* utf8Destination
 ) noexcept;
 
+MSIX_API HRESULT STDMETHODCALLTYPE UnpackPackageFromPackageReader(
+    IAppxPackageReader* packageReader,

```

Comment:
The reader should come after the options for consistency.

Reply:
But before the destination... for consistency.

---
In reply to: [291763404](https://github.com/Microsoft/msix-packaging/pull/152#discussion_r291763404) [](ancestors = 291763404)